pkg/chartvalues/credentiald.go

```diff
@@ -0,0 +1,57 @@
+package chartvalues
+
+import (
+	"reflect"
+
+	"github.com/giantswarm/e2etemplates/internal/render"
+	"github.com/giantswarm/microerror"
+)
+
+type CredentialdConfig struct {
+	AWS                CredentialdConfigAWS
+	Azure              CredentialdConfigAzure
+	Deployment         CredentialdConfigDeployment
+	RegistryPullSecret string
+}
+
+type CredentialdConfigAWS struct {
+	CredentialDefault CredentialdConfigAWSCredentialDefault
+}
+
+type CredentialdConfigAWSCredentialDefault struct {
```

Comment:
This isn't a criticism, just learning more about how this works. Why isn't `AdminARN` needed here? https://github.com/giantswarm/e2etemplates/blob/master/pkg/chartvalues/aws_operator.go#L40  Also, should `CredentialDefault` be removed from the `aws_operator` `chartvalues`?

Reply:
`AdminARN` isn't used by aws-operator. This ARN is used by `opsctl open -a cloudprovider`.

Re second part: you are right. I'll remove that. Thank you.

Reply:
Got it. Thanks.